packages/flutter/lib/src/widgets/layout_builder.dart

```diff
@@ -247,6 +247,62 @@ mixin RenderConstrainedLayoutBuilder<ConstraintType extends Constraints, ChildTy
 /// in an [Align] widget. If the child might want to be bigger, consider
 /// wrapping it in a [SingleChildScrollView] or [OverflowBox].
 ///
+/// {@tool dartpad --template=stateless_widget_material}
+///
+/// This example uses a [LayoutBuilder] to build a different widget depending on the available width. Resize the
+/// DartPad window to see [LayoutBuilder] in action!
+/// 
+/// ```dart
+/// class BasicLayoutBuilder extends StatelessWidget {
+///   @override
```

Comment:
I forgot to mention, making this a dartpad means we can remove this boilerplate.
```suggestion
```

Reply:
Here's a dartpad example: https://github.com/flutter/flutter/blob/e64fe11542238376fdfbf351d81e22d08296da45/packages/flutter/lib/src/widgets/basic.dart#L4563 :)